lib/codegen/generator.dart

```diff
@@ -292,7 +292,7 @@ class _DeclarationCode extends ConstExpression {
 
   List<String> get librariesUsed => []
     ..addAll(type.librariesUsed)
-    ..addAll(annotations.expand((a) => a.librariesUsed));
+    ..addAll(annotations.expand((a) => a.librariesUsed) as Iterable<String>);
```

Comment:
Also it is worth noting that without the generic the cast will fail at runtime. The allocated iterable is Iterable<dynamic> which cannot be cast into an Iterable<String> in strong mode.

The alternative here would be to copy the list or iterate over the list by hand so you can cast each individual element.